sklearn/svm/classes.py

```diff
@@ -491,6 +491,14 @@ class SVC(BaseSVC):
     intercept_ : array, shape = [n_class * (n_class-1) / 2]
         Constants in decision function.
 
+    fit_status_ : int
+        0 if correctly fitted, 1 otherwise (will raise warning)
+
+    probA_, probB_ : array, shape = [n_class * (n_class-1) / 2]
+        parameters used to produce probability estimates from decision values,
+        empty array for probability=False. See the section about scores and
+        probabilities in the SVM section of the User Guide for details.
+
```

Comment:
Can you add it as ``References`` to the docstring? Does it also describe the SVR? Can you check that it is also mentioned in the user guide? Thanks.